packages/fluent/scss/listview/_variables.scss

```diff
@@ -73,6 +73,10 @@ $kendo-listview-item-selected-text: initial !default;
 /// @group listview
 $kendo-listview-item-selected-bg: get-theme-color-var( neutral-20 ) !default;
 
+/// Text color of disabled listview.
+/// @group listview
+$kendo-listview-disabled-text: var( --kendo-disabled-text, initial ) !default;
+
```

Comment:
These styles are not needed. The ListView component API does not provide a way to disable the component.